### ISSUE TITLE
Feature request: send/share credential

### ISSUE THREAD

**Issue by gabriele-v** [PhilippC/keepass2android#1379]
Sometimes I have to enter a password in a different computer where I don't have Keepass, but I generally always have my phone with me.

Ideally I'm thinking about something that from Keepass2Android "shares" the credential to a webservice and then on the other PC I access a website (with 2FA) that show only that specific credential and only for a short amount of time.

I could take care of the webservice/website, but I don't know how to do on Keepass2Android side, probably a plugin will be enough, but I've never coded for Android, I guess it could be a simply "PUT" command to the webservice..

**Comment by PhilippC** [comment]
you might want to try https://play.google.com/store/apps/details?id=mzrw.k2aplugin.bluetoothkeyboard ?

**Comment by gabriele-v** [comment]
Yep something like this, but Bluetooth it's not applicable because for example it's missing from most of desktop PCs
